scripts/build_and_distribute.sh

```
@@ -0,0 +1,179 @@
+if [ ! -f "$ENV_FILE" ]; then
+    echo "❌ Error: scripts/.env file not found!"
+    echo ""
+    echo "Create scripts/.env file with the following content:"
+    echo "export FIREBASE_API_KEY=\"your-api-key\""
+    echo "export FIREBASE_GCM_SENDER_ID=\"your-sender-id\""
+    echo "export FIREBASE_APP_ID=\"your-app-id\""
+    exit 1
+fi
+
+# Source the .env file to load environment variables
+source "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)